ttnn/cpp/ttnn/operations/moreh/moreh_linear_backward/device/moreh_linear_backward_device_operation.hpp

```diff
@@ -0,0 +1,94 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <variant>
+#include <vector>
+
+#include "ttnn/decorators.hpp"
+#include "ttnn/device_operation.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+#include "ttnn/tensor/tensor.hpp"
+#include "ttnn/tensor/types.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear_backward {
+struct MorehBiasAddBackwardOperation {
+    struct operation_attributes_t {
+        const MemoryConfig bias_grad_mem_config;
```

Comment:
```suggestion
        const MemoryConfig bias_grad_memory_config;
```

Reply:
solved in [b87ffe7](https://github.com/tenstorrent/tt-metal/pull/12812/commits/b87ffe749acd90b038cdc391f7cb2e0b8e81f3cb)